src/NetworkTest/testQuality/helpers/subscriberMOS.ts

```diff
@@ -37,26 +38,47 @@ function calculateVideoScore(subscriber: OT.Subscriber, stats: OT.SubscriberStat
   const baseBitrate = calculateBitRate('video', currentStats, lastStats);
   const pixelCount = subscriber.stream.videoDimensions.width * subscriber.stream.videoDimensions.height;
   const targetBitrate = targetBitrateForPixelCount(pixelCount);
-
   if (baseBitrate < MIN_VIDEO_BITRATE) {
     return 1;
   }
   const bitrate = Math.min(baseBitrate, targetBitrate);
-
   let score =
     ((Math.log(bitrate / MIN_VIDEO_BITRATE) / Math.log(targetBitrate / MIN_VIDEO_BITRATE)) * 4) + 1;
   score = Math.min(score, 4.5);
   return score;
 }
 
-function calculateAudioScore(subscriber: OT.Subscriber, stats: OT.SubscriberStats[]): number {
+function calculateAudioScore(subscriber: OT.Subscriber, publisherStats: OT.PublisherRtcStatsReportArr | undefined, stats: OT.SubscriberStats[]): number {
```

Comment:
I get a tslint error here: Exceeds maximum line length of 120.

Reply:
Thanks @jeffswartz I just updated this line. Could you please review?